insights/specs/datasources/ethernet.py

```diff
@@ -55,3 +56,32 @@ def interfaces(broker):
             return sorted(ifaces)
 
     raise SkipComponent
+
+
+@datasource(NmcliConnShow, HostContext)
+def team_device(broker):
+    """
+    This datasource provides a list of the team device.
+
+    Sample data returned::
+
+        ['team0', 'team1']
+
+    Returns:
+        list: List of the team device.
+
+    Raises:
+        SkipComponent: When there is not any content.
+    """
+
+    content = broker[NmcliConnShow].data
+    if content:
+        team_device_list = []
+        for x in content:
+            if 'team' in x['TYPE']:
```

Comment:
Would you please double confirm if the `disconnected` team interfaces good candidates to this parser?

Reply:
Updated